packages/react-ui-components/src/components/SchemaForm/Widgets/AdvancedDateSelectDropdownWidget/AdvancedDateSelectDropdownWidget.tsx

```diff
@@ -0,0 +1,51 @@
+import { ICalendarAdvanceSelectedTime, ITimeChange } from '@kelvininc/ui-components';
+import { WidgetProps } from '@rjsf/utils';
+import { get } from 'lodash';
+import React, { useMemo, useState } from 'react';
+import { KvAdvancedDateSelectDropdown, KvFormLabel } from '../../../stencil-generated';
+import { DEFAULT_DROPDOWN_POSITION_OPTIONS } from './config';
+
+const AdvancedDateSelectDropdownWidget = ({ schema, label, id, disabled, required, onChange, uiSchema }: WidgetProps) => {
+	const { inputConfig, selectedTime, absoluteTimeConfig, relativeTimeConfig, selectedTimezone, dateMask, dropdownPositionOptions: options, size } = schema;
+
+	const config = useMemo(
+		() => ({
+			...inputConfig,
+			disabled: inputConfig?.['disabled'] ?? disabled
+		}),
+		[inputConfig, disabled]
+	);
+
+	const dropdownPositionOptions = options ?? DEFAULT_DROPDOWN_POSITION_OPTIONS;
```

Comment:
This could be moved to the schema destructuring.

```
const { inputConfig, selectedTime, absoluteTimeConfig, relativeTimeConfig, selectedTimezone, dateMask, dropdownPositionOptions = DEFAULT_DROPDOWN_POSITION_OPTIONS, size } = schema;
```